__tests__/articles.test.js

```diff
@@ -51,3 +52,46 @@ describe("/api/articles/:article_id", () => {
       });
   });
 });
+
+describe.only("/api/articles", () => {
```

Comment:
Don't leave .only in test files. This gets around the husky pre commit and now I have no confidence other tests are passing.